Makefile

```diff
@@ -1,11 +1,13 @@
 MAKE?=make
 PYTHON?=/usr/bin/env python3
+BLACK_OPTIONS=-l 120 --exclude 'action2var_variables.py|action3_callbacks.py|generated'
```

Comment:
if you like, black options can also be put in `pyproject.toml`, see https://black.readthedocs.io/en/stable/pyproject_toml.html .
This makes sure that if someone executed `black`, it picks up the right rules too.
I have yet to apply this for all other OpenTTD projects :)